File: Dev/CS/Mascaret/Mascaret/VEHA/Kernel/LiteralSpecification.cs

using System;
using System.Collections.Generic;


namespace Mascaret
{
    public abstract class LiteralSpecification : ValueSpecification
    {
        public LiteralSpecification(Classifier type)
            : base(type)
        {
            CLASSTYPE = "Litteral";
        }

        public override ValueSpecification clone()
        {
            return null;
        }

    }
}

File: Dev/C++/Mascaret/src/VEHA/Kernel/LiteralSpecification.cpp
#include "VEHA/Kernel/LiteralSpecification.h"

namespace VEHA
{

LiteralSpecification::LiteralSpecification(shared_ptr<Classifier> type)
:ValueSpecification(type)
{
	CLASSTYPE="Litteral";
}

LiteralSpecification::~LiteralSpecification()
{
}

}
